src/main/java/org/gradle/profiler/flamegraph/FlameGraphSanitizer.java

```diff
@@ -213,6 +215,15 @@ private ChopPrefix(String stopToken) {
         }
     }
 
+    private static class NormalizeLambda extends FrameWiseSanitizeFunction {
+        @Override
+        protected String mapFrame(String frame) {
+            // Lambdas contain a name that's based on an index + timestamp at runtime and changes build-to-build.
+            // This makes comparing two builds very difficult when a lambda is in the stack
+            return frame.replaceFirst(Pattern.quote("$$Lambda$")+"[0-9/]+", "\\$\\$Lambda\\$");
```

Comment:
```suggestion
            return frame.replaceFirst(Pattern.quote("$$Lambda$")+"[0-9./]+", "\\$\\$Lambda\\$");
```

Can you leave the `.` and the `/`? That way we also normalize Lambdas for different Java versions correctly.